.github/CODEOWNERS

```
@@ -1,2 +1,2 @@
 * @az-digital/maintainers
-* @az-digital/developers
+
```

As discussed last week, I don't think we want to make this change.  This sets the default reviews for new PRs.
```suggestion
* @az-digital/developers
```